Raven.Tests.Helpers/RavenTestBase.cs

```diff
@@ -865,5 +962,65 @@ protected static void DeployNorthwind(DocumentStore store, string databaseName =
 			var request = requestFactory.CreateHttpJsonRequest(new CreateHttpJsonRequestParams(null, url + "/studio-tasks/createSampleData", "POST", store.DatabaseCommands.PrimaryCredentials, store.Conventions));
 			request.ExecuteRequest();
 		}
+
+        private static void StoreSeedData(IEnumerable<IEnumerable> seedData,
```

Comment:
This method signature is interesting. Right now it's designed to take a collection of collections:

eg. 
```
var seedData = new IEnumerable[]
        {
            FakeModelHelpers.CreateFakeCats(),
            FakeModelHelpers.CreateFakeDogs()
        };
```

But i was wondering if it might be kewl to make it accept just a single object and I try and see if that object is `Enumerable` and if so, iterate through it? (and do that 2x - first for the top collection, 2nd check for the items in each collection).

Why? just to make it easier for a developer to consume this method.
eg.
```
var seedData = new Cat();
var seedData = CreateFakeCats(); // IEnumerable<Cat>
var seedData = new IEnumerable[] { .. // as above in the top of this comment ... };
```